contracts/sol6/KyberStorage.sol

```diff
@@ -585,6 +633,22 @@ contract KyberStorage is IKyberStorage, PermissionGroupsNoModifiers, Utils5 {
             // if reserve wasn't found add it
             reserveArr.push(reserveId);
         }
+
+        for (i = 0; i < tokensArr.length; i++) {
```

Comment:
I think here !add should be the external one.
so we don't loop if not removing.

Reply:
so actually if add
just push (don't check tokensArr length.

Else
do the loop and remove